lms/envs/common.py

```diff
@@ -3127,6 +3127,7 @@
 
 ENTERPRISE_ENROLLMENT_API_URL = LMS_ROOT_URL + "/api/enrollment/v1/"
 ENTERPRISE_PUBLIC_ENROLLMENT_API_URL = ENTERPRISE_ENROLLMENT_API_URL
+ENTERPRISE_COURSE_ENROLLMENT_AUDIT_MODES = ['audit']
```

Comment:
This new setting is not documented anywhere - please add some comments explaining what it does (ideally in edx-enterprise too).